GVFS/GVFS.Installer.Mac/CreateMacInstaller.sh

```diff
@@ -70,10 +78,19 @@ function CopyBinariesToInstall()
     copyPrjFS="cp -Rf \"${VFS_OUTPUTDIR}/ProjFS.Mac/Native/$CONFIGURATION\"/PrjFSKext.kext \"${STAGINGDIR}/${PRJFSDESTINATION}/.\""
     eval $copyPrjFS || exit 1
     
+    copyPrjFS="cp -Rf \"${VFS_OUTPUTDIR}/ProjFS.Mac/Native/$CONFIGURATION\"/PrjFSKext.kext \"${STAGINGDIR}/${LIBRARYEXTENSIONSDESTINATION}/.\""
```

Comment:
Why does `PrjFSKext.kext ` need to get copied to two locations?

Reply:
Possibly answering my own question, is this so the kext is auto-loaded?

Reply:
If we're installing to `/Library/Extensions` I don't see any reason to also install it somewhere else.

Reply:
Removed the Kext copy inside `/usr/local/vfsforgit/prjfs`